Assets/Scripts/Game/Player/PlayerHeightChanger.cs

```diff
@@ -119,7 +119,7 @@ public void DecideHeightAction()
                 heightAction = HeightChangeAction.DoDismounting;
                 toggleRiding = false;
             }
-            else if (!playerMotor.IsRiding)
+            else if (!playerMotor.IsRiding && !playerMotor.IsCrouching)
```

Comment:
I've noticed player can no longer stand back up after this change. Once they crouch, they're stuck crouching. This happens because the condition only allows crouch toggle when player is not already crouch. I think the intention was to prevent crouch toggle while climbing, rather than when already crouched?

I've reverted this change now so player can toggle crouch again. At this time, they will simply fall if trying to change stance while climbing.